alerting/src/main/kotlin/org/opensearch/alerting/model/MonitorMetadata.kt

```diff
@@ -64,6 +65,16 @@ data class MonitorMetadata(
         return builder.endObject()
     }
 
+    fun addQueryIndexToMapping(sourceIndex: String, queryIndex: String) {
+        val index = IndexUtils.sanitizeDotsInIndexName(sourceIndex)
```

Comment:
This is needed only because we removed "enabled: false" from scheduled_job index schema.